apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v2_0/extensions/RoleAdminApi.java

```diff
@@ -48,7 +48,7 @@
  *
  */
 @Beta
-@Extension(of = ServiceType.IDENTITY, namespace = ExtensionNamespaces.OS_KSADM)
+@Extension(of = ServiceType.IDENTITY, namespace = ExtensionNamespaces.OS_KSADM, name = "OpenStack Keystone Admin", alias = "OS-KSADM")
```

Comment:
Keystone only has two extensions, and I wasn't sure how I wanted these implemented. Considering how nova is, I will probably add another file and refactor these to constants.